### PR TITLE
fix code

### PR DESCRIPTION
**Pull request by HiQoSolutionsAI**

### CODE REVIEW

File: src/app/header/header.component.html [hunk 1]
```diff
@@ -1,24 +1,17 @@
...
-      class="user-image"
-      src="{{ (user | async)?.profilePhoto }}"
-      alt="user photo"
-    />
-    <button
-      mat-button
-      class="user-name mat-body-1"
-      [matMenuTriggerFor]="userMenu"
-    >
+    <img class="user-image" src="{{ (user | async)?.profilePhoto }}" alt="user photo" />
```
Comment (EkaterinaSaverina) [inline src/app/header/header.component.html]: can be improved

File: src/app/auth/auth.component.scss [hunk 2]
```diff
@@ -1,5 +1,5 @@
-@import "mixins";
-@import "variables";
+@import "_mixins";
+@import "_variables";
```
Comment (EkaterinaSaverina) [inline src/app/auth/auth.component.scss]: Do we need _ in imports?

File: src/app/auth/auth.component.spec.ts [hunk 3]
```diff
@@ -1,18 +1,17 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 
-import { LoginComponent } from './login.component';
+import { AuthComponent } from './auth.component';
 
 describe('LoginComponent', () => {
```
Comment (EkaterinaSaverina) [inline src/app/auth/auth.component.spec.ts]: One more string to rename

File: src/app/auth/auth.module.ts [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
 import { SharedModule } from '@app/shared/shared.module';
 
 @NgModule({
-  declarations: [LoginComponent],
+  declarations: [AuthComponent],
   imports: [CommonModule, LoginRoutingModule, SharedModule],
-  exports: [LoginComponent],
+  exports: [AuthComponent],
 })
 export class LoginModule {}
```
Comment (EkaterinaSaverina) [inline src/app/auth/auth.module.ts]: the same

File: src/app/header/header.component.ts [hunk 5]
```diff
@@ -5,18 +5,22 @@ import { Observable } from 'rxjs';
...
   templateUrl: './header.component.html',
   styleUrls: ['./header.component.scss'],
 })
 export class HeaderComponent {
-  user: Observable<User> | undefined = this.store.select(selectGetUserAuth);
+  user$: Observable<User> | undefined = this.store.select(userAuthSelector.selectGetUserAuth);
 
   constructor(private auth: AuthService, private router: Router, private store: Store) {}
 
+  switchToDashboard() {
```
Comment (EkaterinaSaverina) [inline src/app/header/header.component.ts]: Add a return type

File: src/app/not-found/not-found.component.ts [hunk 6]
```diff
@@ -3,13 +3,8 @@ import { Component, OnInit } from '@angular/core';
...
 })
-export class NotFoundComponent implements OnInit {
-
-  constructor() { }
-
-  ngOnInit(): void {
-  }
-
+export class NotFoundComponent {
+  constructor() {}
```
Comment (EkaterinaSaverina) [inline src/app/not-found/not-found.component.ts]: No need to leave an empry constructor

File: src/scss/mixins/_buttons.scss [hunk 7]
```diff
@@ -0,0 +1,5 @@
+@mixin button($color: $color-basic, $bg-color: $color-primary, $text-transform: uppercase) {
+  color: $color;
+  background-color: $bg-color;
+  text-transform: $text-transform;
+}
```
Comment (EkaterinaSaverina) [inline src/scss/mixins/_buttons.scss:5]: Can be improved by adding states on hover, active, disabled or focus

File: src/scss/mixins/_inputs.scss [hunk 8]
```diff
@@ -0,0 +1,7 @@
+@mixin input($color: $color-basic, $bg-color: $color-primary) {
+  color: $color;
+  background-color: $bg-color;
+  outline: none;
+  border: none;
+  border-bottom: 1px solid $color-basic;
+}
```
Comment (EkaterinaSaverina) [inline src/scss/mixins/_inputs.scss]: Please use Form field and Input material components instead of default input

File: src/styles.scss [hunk 9]
```diff
@@ -1,7 +1,14 @@
...
+@import "components";
+@import "typography";
 
-html, body { height: 100%; }
-body { margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; }
+html,
+body {
+  height: 100%;
+}
+body {
```
Comment (EkaterinaSaverina) [inline src/styles.scss:13]: miss an empty line

File: src/scss/mixins/_buttons.scss (hunk 7 above)
Comment (HiQoSolutionsAI) [reply src/scss/mixins/_buttons.scss:5]: I use these styles from Angular material

File: src/scss/_utilities.scss [hunk 10]
```diff
@@ -0,0 +1,8 @@
+@each $value in 5, 10, 15, 20, 25 {
+  .mt-#{$value} {
+    margin-top: #{$value}px !important;
```
Comment (EkaterinaSaverina) [inline src/scss/_utilities.scss:3]: Do we need important here?

File: src/app/board/list/list.component.html [hunk 11]
```diff
@@ -9,7 +9,8 @@ <h4 class="list-title" (click)="toggleTitleEditMode()">{{ listTitle }}</h4>
 <ng-template #editTitle>
   <input
     #inputTitle
-    class="edit-input list-input"
+    matInput
```
Comment (EkaterinaSaverina) [inline src/app/board/list/list.component.html]: matInput should be wrapped by mat form field

File: src/app/dashboard/main-boards/main-boards.component.html [hunk 12]
```diff
@@ -28,7 +29,8 @@ <h4 class="main-board-title" (click)="toggleTitleEditMode()">{{ boardTitle }}</h
 <ng-template #editDescription>
   <input
     #inputDescription
-    class="edit-input main-board-input"
+    matInput
+    class="main-board-input mt-5 mb-5"
```
Comment (EkaterinaSaverina) [inline src/app/dashboard/main-boards/main-boards.component.html]: can be improved: if we need the same value for margin we can add a single class in utils for it

File: src/app/auth/auth.component.scss [hunk 13]
```diff
@@ -75,8 +75,5 @@
 }
 
 .button-login {
-  text-transform: uppercase;
-  color: $color-basic;
-  background-color: $color-primary;
   margin-top: 25px;
```
Comment (EkaterinaSaverina) [inline src/app/auth/auth.component.scss]: please use a class from utils

File: src/scss/_utilities.scss [hunk 14]
```diff
@@ -5,4 +5,18 @@
...
+  .mr-#{$value} {
+    margin-right: #{$value}px;
+  }
+  .ml-mr-#{$value} {
+    margin-left: #{$value}px;
+    margin-right: #{$value}px;
+  }
+  .mt-mb-#{$value} {
+    margin-top: #{$value}px;
+    margin-bottom: #{$value}px;
```
Comment (EkaterinaSaverina) [inline src/scss/_utilities.scss]: you can call it just mx and my
like in bootstrap

**Review by EkaterinaSaverina — APPROVED** [review]

**Pull request merged**